independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/commands/CreateProjectHelper.java

```diff
@@ -34,7 +34,7 @@ public class CreateProjectHelper {
 
     // ordering is important here, so let's keep them ordered
     public static final SortedSet<Integer> JAVA_VERSIONS_LTS = new TreeSet<>(List.of(11, 17));
-    private static final int DEFAULT_JAVA_VERSION = 11;
+    private static final int DEFAULT_JAVA_VERSION = 17;
```

Comment:
I'm pretty sure it's going to break `determineBestJavaLtsVersion`. The current situation here is to create a project with the highest LTS version supported by the runtime used to create the project.

I think we should keep it this way.

Reply:
Ok then you can close!